Lib/test/test_capi.py

```diff
@@ -272,6 +272,50 @@ class C(): pass
         self.assertIn(b'MemoryError 2 20', out)
         self.assertIn(b'MemoryError 3 30', out)
 
+    def test_mapping_keys_values_items(self):
+        class Mapping1(dict):
```

Comment:
Is there a need to test this class?

Reply:
Its purpose is to test cases in which the methods already return a `list`, i.e. when `PyList_CheckExact(meth_output)` is true.